src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -253,58 +256,6 @@ public virtual Task DispatchEventAsync(int eventHandlerId, EventFieldInfo fieldI
             return GetErrorHandledTask(task);
         }
 
-        /// <summary>
-        /// Executes the supplied work item on the renderer's
-        /// synchronization context.
-        /// </summary>
-        /// <param name="workItem">The work item to execute.</param>
-        public virtual Task InvokeAsync(Action workItem)
```

Comment:
A general thread in this PR is removing "helper" code like this and folding the functionality into the dispatcher directly.